activesupport/lib/active_support/message_encryptor.rb

```diff
@@ -22,6 +22,28 @@ module ActiveSupport
   #   crypt = ActiveSupport::MessageEncryptor.new(key)                            # => #<ActiveSupport::MessageEncryptor ...>
   #   encrypted_data = crypt.encrypt_and_sign('my secret data')                   # => "NlFBTTMwOUV5UlA1QlNEN2xkY2d6eThYWWh..."
   #   crypt.decrypt_and_verify(encrypted_data)                                    # => "my secret data"
+  #
+  # MessageEncryptor supports the following metadata that's associated with a message:
+  #
+  # * <tt>+:purpose+</tt> - The purpose for which the message is intended to be used.
+  # * <tt>+:expires_at+</tt> - The expiration date of the message.
+  # * <tt>+:expires_in+</tt> - The relative expiration date. Note that, :expires_at takes
+  #                          precedence over :expires_in.
```

Comment:
Could you align the beginning of this line with the beginning of the above one ? Otherwise, RDoc will think that this is a code snippet.

You can check-out how the documentation you write will look on the API site running `bundle exec rake rdoc` and by opening the `index.html` file under `doc/rdoc`. :-)